charts/sysdig/README.md

```diff
@@ -167,6 +168,38 @@ $ helm install --namespace sysdig-agent sysdig-agent -f values.yaml sysdig/sysdi
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+## Resource profiles
+For ease of use, three predefined resource profiles are available:
+* small
+```yaml
+requests:
+  cpu: 1000m
+  memory: 1024Mi
+limits:
+  cpu: 1000m
+  memory: 1024Mi
+```
+* medium
+```yaml
+requests:
+  cpu: 3000m
+  memory: 3072Mi
+limits:
+  cpu: 3000m
+  memory: 3072Mi
+```
+* large
+```yaml
+requests:
+  cpu: 5000m
+  memory: 6144Mi
+limits:
+  cpu: 5000m
+  memory: 6144Mi
+```
+* custom
+  By setting "custom" or any value other than the ones defined above, you will be able to set custom values using `resources` object.
```

Comment:
Can we say instead "You can create your own custom profile to match your requirements by setting the appropriate values for the custom profile in resources object.